coalib/processes/Processing.py

```diff
@@ -210,6 +211,7 @@ def print_result(results,
                            diff objects as values.
     :param ignore_ranges:  A list of SourceRanges. Results that affect code in
                            any of those ranges will be ignored.
+    :param console_printer:Object to print messages on the console.
```

Comment:
I think you'll have to indent all the comments here by one space, having no space after colon isn't good